docs/resources/resources.md

```diff
@@ -7,6 +7,9 @@ title:  "Resources"
 
 _These articles reflect the state of ZIO at the time of their publication. The code samples might be outdated, considering ZIO was early in development at the time they were written. However, the concepts are still relevant._
 
+- [Simulating IoT Events with Zio Streams](https://timpigden.github.io/_pages/zio-streams/GeneratingChillEvents.html) by Tim Pigden (November 2019)
```

Comment:
This is very minor but I think the common spelling is all caps: ZIO. 

Very nice articles by the way!